src/d3d9/d3d9_depth_bias.cpp

```diff
@@ -0,0 +1,200 @@
+    }
+  }
+
+  uint32_t D3D9DepthBias::DetermineFixedFactor(VkFormat Format) {
+    DxvkImageCreateInfo dsCreateInfo;
+    dsCreateInfo.type = VK_IMAGE_TYPE_2D;
+    dsCreateInfo.format = Format;
+    dsCreateInfo.flags = 0;
+    dsCreateInfo.sampleCount = VK_SAMPLE_COUNT_1_BIT;
+    dsCreateInfo.extent = { 5, 1, 1};
+    dsCreateInfo.numLayers = 1;
+    dsCreateInfo.mipLevels = 1;
+    dsCreateInfo.usage = VK_IMAGE_USAGE_DEPTH_STENCIL_ATTACHMENT_BIT | VK_IMAGE_USAGE_TRANSFER_SRC_BIT;
+    dsCreateInfo.stages = VK_PIPELINE_STAGE_EARLY_FRAGMENT_TESTS_BIT | VK_PIPELINE_STAGE_LATE_FRAGMENT_TESTS_BIT | VK_PIPELINE_STAGE_TRANSFER_BIT;
+    dsCreateInfo.access = VK_ACCESS_DEPTH_STENCIL_ATTACHMENT_READ_BIT | VK_ACCESS_DEPTH_STENCIL_ATTACHMENT_WRITE_BIT | VK_ACCESS_TRANSFER_READ_BIT;
+    dsCreateInfo.tiling = VK_IMAGE_TILING_OPTIMAL;
+    dsCreateInfo.layout = VK_IMAGE_LAYOUT_DEPTH_STENCIL_ATTACHMENT_OPTIMAL;
+    Rc<DxvkImage> ds = m_device->createImage(dsCreateInfo, VK_MEMORY_PROPERTY_DEVICE_LOCAL_BIT);
+
+    DxvkImageViewCreateInfo dsvCreateInfo;
+    dsvCreateInfo.usage = VK_IMAGE_USAGE_DEPTH_STENCIL_ATTACHMENT_BIT;
+    dsvCreateInfo.numLevels = 1;
+    dsvCreateInfo.numLayers = 1;
+    dsvCreateInfo.aspect = VK_IMAGE_ASPECT_DEPTH_BIT;
+    dsvCreateInfo.format = Format;
+    Rc<DxvkImageView> dsv = m_device->createImageView(ds, dsvCreateInfo);
+
+    m_context->beginRecording(m_device->createCommandList());
+
+    VkViewport viewport;
+    viewport.x        = 0.5f; // Shift viewport so 0,0 is sampled at the top left of the pixel
+    viewport.y        = 0.0f;
+    viewport.width    = 5.0f;
+    viewport.height   = 1.0f;
+    viewport.minDepth = 0.0f;
+    viewport.maxDepth = 1.0f;
+    VkRect2D scissor;
+    scissor.offset    = { 0, 0   };
+    scissor.extent    = { 5, 1 };
+    m_context->setViewports(1, &viewport, &scissor);
+
+    DxvkDepthBias biases;
+    biases.depthBiasConstant = 1.0f;
+    biases.depthBiasSlope    = 0.0f;
+    biases.depthBiasClamp    = 0.0f;
+    m_context->setDepthBias(biases);
+
+    DxvkDepthBounds bounds;
+    bounds.enableDepthBounds = VK_TRUE;
+    bounds.minDepthBounds = 0.0f;
+    bounds.maxDepthBounds = 1.0f;
+    m_context->setDepthBounds(bounds);
+
+    VkClearValue clearValue;
+    clearValue.depthStencil.depth = 0;
+    clearValue.depthStencil.stencil = 0;
+    m_context->clearRenderTarget(dsv, VK_IMAGE_ASPECT_DEPTH_BIT | VK_IMAGE_ASPECT_STENCIL_BIT, clearValue);
+
+    DxvkRenderTargets rts = { };
+    rts.depth.view = dsv;
+    rts.depth.layout = VK_IMAGE_LAYOUT_DEPTH_STENCIL_ATTACHMENT_OPTIMAL;
+    m_context->bindRenderTargets(std::move(rts), 0);
+
+    DxvkVertexAttribute attribute = { 0, 0, VK_FORMAT_R32G32B32A32_SFLOAT, 0};
+    DxvkVertexBinding binding = { 0, 0, VK_VERTEX_INPUT_RATE_VERTEX, 16 };
+    m_context->setInputLayout(1, &attribute, 1, &binding);
+
+    DxvkRasterizerState rsState;
+    rsState.polygonMode = VK_POLYGON_MODE_FILL;
+    rsState.cullMode = VK_CULL_MODE_NONE;
```

Comment:
Would be nice to line up the = signs (use "Align" in vs code to do automatically)